examples/splinterdb_custom_ipv4_addr_sortcmp_example.c

```diff
@@ -29,18 +29,13 @@
 
 #include "splinterdb/default_data_config.h"
 #include "splinterdb/splinterdb.h"
-#include "example_common.h"
-
-/* Tag to identify messages from application program */
-#define APP_ME "App-IPV4-sortcmp"
 
 /*
  * App-specific 'defaults' that can be parameterized, eventually.
  */
-#define APP_DB_NAME "splinterdb_custom_ipv4_sortcmp_example_db"
-
-#define APP_DEVICE_SIZE_MB 1024 // Size of SplinterDB device; Fixed when created
-#define APP_CACHE_SIZE_MB  64   // Size of cache; can be changed across boots
+#define DB_FILE_NAME    "splinterdb_custom_ipv4_sortcmp_example_db"
+#define DB_FILE_SIZE_MB 1024 // Size of SplinterDB device; Fixed when created
+#define CACHE_SIZE_MB   64   // Size of cache; can be changed across boots
```

Comment:
Consistently reworked above 3 names (as suggested in previous review meeting). This change is now applied to all programs.